src/askscript/__tests__/90-misc/fetch.test.ts

```diff
@@ -0,0 +1,6 @@
+import program from './fetch.ask';
+
+test('fetch', async () => {
+  const json = await program();
+  expect(json.id).toBe(1);
+});
```

Comment:
@RichardM99 can we mock the endpoint for tests? The tests should run in isolation. You can find more about mocking in Jest here https://jestjs.io/docs/en/mock-functions

Reply:
It's also useful to hit the real endpoint, however, that would be an integration test for Node which we don't yet have - we currently focus on unit test coverage

Reply:
I'm going to let @czerwinskilukasz1 to continue guide you through your first contributions :)